packages/types/src/event.ts

```diff
@@ -47,7 +47,7 @@ export interface Event {
 }
 
 /** JSDoc */
-export type EventType = 'transaction';
+export type EventType = 'event' | 'transaction';
```

Comment:
I thought this EventType refer to Error, Transaction, Default? Which unfortunately might not match envelope header requirements